src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaRepository.java

```diff
@@ -99,6 +94,30 @@ public void createTopic(final String topic, final int partitionsNum, final int r
         }
     }
 
+    public boolean topicExists(final String topic) throws NakadiException {
```

Comment:
Then we should delete the `Topic` class?
Anyway, for me it's still the name of a topic. A topic consists of the name, the partitions, lots of events, a configuration etc. It already confused me multiple times when reading the code.